### PR TITLE
Depositor bot guide

### PR DESCRIPTION
**Pull request by hweawer**

### CODE REVIEW

File: docs/guides/depositor-bot.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
+# Lido depositor bot
+
+## Introduction
+
+Depositor bot is a part of Deposit Security Module.
```
Comment (F4ever) [inline docs/guides/depositor-bot.md:5]:
```suggestion
Depositor bot is a part of [Deposit Security Module](link to dsm).
```

File: docs/guides/depositor-bot.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+The Depositor Bot obtains signed deposit messages from Council Daemons. Once a sufficient number of messages is
+collected to constitute a quorum, the bot proceeds to initiate a deposit into the designated staking module. This
+deposit is executed using the depositBufferedEther function within the "DepositSecurityModule" smart contract.
```
Comment (F4ever) [inline docs/guides/depositor-bot.md:9]: link to the function spec

File: docs/guides/depositor-bot.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+### Hardware
+
+- 1-core CPU
+- 2GB RAM
+
+### Nodes
+
+- Ethereum EL node
+- Onchain transport node
```
Comment (F4ever) [inline docs/guides/depositor-bot.md:21]:
```suggestion
- Onchain databus transport node
```

Comment (F4ever) [reply docs/guides/depositor-bot.md:21]: Add comment that currently it's gnosis chain

File: docs/guides/depositor-bot.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+
+### Envs
+
+Required variables are:
```
Comment (F4ever) [inline docs/guides/depositor-bot.md:25]: Add note that defaults is correct for mainnet

File: docs/guides/depositor-bot.md [hunk 5]
```diff
@@ -0,0 +1,74 @@
+# Lido depositor bot
+
+## Introduction
+
+Depositor bot is a part of Deposit Security Module.
+
+The Depositor Bot obtains signed deposit messages from Council Daemons. Once a sufficient number of messages is
```
Comment (F4ever) [inline docs/guides/depositor-bot.md:7]: Killer feature of deposit bot is gas price checking add some details how it works.
Also add to envs list gas price envs

File: docs/guides/depositor-bot.md (hunk 1 above)
Comment (hweawer) [reply docs/guides/depositor-bot.md]: Done

File: docs/guides/depositor-bot.md (hunk 2 above)
Comment (hweawer) [reply docs/guides/depositor-bot.md]: Done

File: docs/guides/depositor-bot.md (hunk 3 above)
Comment (hweawer) [reply docs/guides/depositor-bot.md]: Done

File: docs/guides/depositor-bot.md (hunk 4 above)
Comment (hweawer) [reply docs/guides/depositor-bot.md]: Done

File: docs/guides/depositor-bot.md [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+    {"name": "bots.depositor", "levelname": "INFO", "funcName": "_deposit_to_module", "lineno": 194, "module": "depositor", "pathname": "/app/src/bots/depositor.py", "timestamp": 1729511569, "msg": "Calculations deposit recommendations.", "value": false, "is_mellow": false}
+    ```
+  
+  If you are facing problems, check what environment variables depositor bot is using, find a log line `"msg": "Bot env variables"`
+
+### Docker
+
+Docker image could be found [here](https://docs.lido.fi/guides/tooling#depositor-bot).
+
+## Monitoring
```
[the remark this answers is not in the document]
Comment (hweawer) [reply docs/guides/depositor-bot.md:75]: Added examples in https://github.com/lidofinance/depositor-bot/pull/276

File: docs/guides/depositor-bot.md (hunk 5 above)
Comment (hweawer) [reply docs/guides/depositor-bot.md:7]: Added

File: docs/guides/depositor-bot.md [hunk 7]
```diff
@@ -0,0 +1,104 @@
+# Lido depositor bot
+
+## Introduction
+
+Depositor bot is a part of [Deposit Security Module](link to dsm)..
```
Comment (TheDZhon) [inline docs/guides/depositor-bot.md:5]:
```suggestion
Depositor bot is a part of [Deposit Security Module](/contracts/deposit-security-module).
```

File: docs/guides/depositor-bot.md (hunk 5 above)
Comment (TheDZhon) [reply docs/guides/depositor-bot.md:7]: +to @F4ever 
see no mentions of this as of now

File: docs/guides/depositor-bot.md [hunk 8]
```diff
@@ -0,0 +1,104 @@
...
+the [DepositSecurityModule](https://docs.lido.fi/contracts/deposit-security-module) smart contract.
+
+## Requirements
+
+### Hardware
+
+- 1-core CPU
+- 2GB RAM
+
+### Nodes
```
Comment (TheDZhon) [inline docs/guides/depositor-bot.md:20]: Does it really need nodes, or would RPC services be enough? :thinking:

File: docs/guides/depositor-bot.md [hunk 9]
```diff
@@ -0,0 +1,104 @@
...
+|-----------------------------------|--------------------------------------------|--------------------------------------------------------------------------------------------------------------------------|
+| WEB3_RPC_ENDPOINTS                | -                                          | List of rpc endpoints that will be used to send requests comma separated (`,`)                                           |
+| WALLET_PRIVATE_KEY                | -                                          | Account private key                                                                                                      |
+| CREATE_TRANSACTIONS               | false                                      | If true then tx will be send to blockchain                                                                               |
+| LIDO_LOCATOR                      | 0xC1d0b3DE6792Bf6b4b37EccdcC24e45978Cfd2Eb | Lido Locator address. Mainnet by default. Other networks could be found [here](https://docs.lido.fi/deployed-contracts/) |
```
Comment (TheDZhon) [inline docs/guides/depositor-bot.md:54]:
```suggestion
| LIDO_LOCATOR                      | 0xC1d0b3DE6792Bf6b4b37EccdcC24e45978Cfd2Eb | Lido Locator address. Mainnet by default. Other networks can be found [here](https://docs.lido.fi/deployed-contracts/) |
```

File: docs/guides/depositor-bot.md [hunk 10]
```diff
@@ -0,0 +1,104 @@
...
+maxPriorityFeePerGas = priority
+```
+
+### Envs
+
+Required variables are(mainnet):
+
+| Variable                          | Default                                    | Description                                                                                                              |
+|-----------------------------------|--------------------------------------------|--------------------------------------------------------------------------------------------------------------------------|
+| WEB3_RPC_ENDPOINTS                | -                                          | List of rpc endpoints that will be used to send requests comma separated (`,`)                                           |
```
Comment (TheDZhon) [inline docs/guides/depositor-bot.md:51]: not clear, tbh, what are the endpoints — both EL and Gnosis or what is the expected format?

File: docs/guides/depositor-bot.md [hunk 11]
```diff
@@ -0,0 +1,104 @@
...
+    {"name": "bots.depositor", "levelname": "INFO", "funcName": "_deposit_to_module", "lineno": 210, "module": "depositor", "pathname": "/app/src/bots/depositor.py", "timestamp": 1729511569, "msg": "Checks failed. Skip deposit."}
+    {"name": "bots.depositor", "levelname": "INFO", "funcName": "_deposit_to_module", "lineno": 194, "module": "depositor", "pathname": "/app/src/bots/depositor.py", "timestamp": 1729511569, "msg": "Calculations deposit recommendations.", "value": false, "is_mellow": false}
+    ```
+
+If you are facing problems, check what environment variables depositor bot is using, find a log
+line `"msg": "Bot env variables"`
+
+### Docker
+
+Docker image could be found [here](https://docs.lido.fi/guides/tooling#depositor-bot).
```
Comment (TheDZhon) [inline docs/guides/depositor-bot.md:100]:
```suggestion
Docker image can be found [here](https://docs.lido.fi/guides/tooling#depositor-bot).
```

**Review by TheDZhon — CHANGES_REQUESTED** [review]
:+1: Thank you!

Left a few suggestions to improve the content a bit!

File: docs/guides/depositor-bot.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (hweawer) [reply docs/guides/depositor-bot.md:102]: Added link

File: docs/guides/depositor-bot.md (hunk 5 above)
Comment (hweawer) [reply docs/guides/depositor-bot.md:7]: @TheDZhon please check `How to use` section

File: docs/guides/depositor-bot.md (hunk 8 above)
Comment (hweawer) [reply docs/guides/depositor-bot.md:20]: Changed

File: docs/guides/depositor-bot.md (hunk 10 above)
Comment (hweawer) [reply docs/guides/depositor-bot.md]: Added in the description

**Review by TheDZhon — APPROVED** [review]
:+1:

**Review by F4ever — APPROVED** [review]

**Review by kadmil — APPROVED** [review]
LGTM

**Pull request merged**